src/nvim/api/vim.c

```diff
@@ -299,7 +299,7 @@ ArrayOf(String) nvim_list_runtime_paths(void)
     FUNC_API_SINCE(1)
 {
   Array rv = ARRAY_DICT_INIT;
-  uint8_t *rtp = p_rtp;
+  char_u *rtp = p_rtp;
```

Comment:
what is the purpose of this change? If we plan to eliminate char_u is uint8_t not the appropriate replacement? Is the concern about systems with char size != 8?